pos_empty_home/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,13 @@
+This module extends Odoo Point Of Sale features, displaying an empty home page
+at the start of the Point of Sale, or when no search is done and no category
+is selected. (when root category is selected)
+
+This module is interesting to avoid to display useless products, when user did
+not yet realized a search. (Default displayed products are just the first
+eighty products)
+
+It can limits errors to avoid users to select undesired product.
+
+
+.. image:: /pos_empty_home/static/description/pos_empty_home.png
```

Comment:
Replace by ../static/description/pos_empty_home.png

This path will work in all cases.